independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/parameters/converters/CharacterParamConverter.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.resteasy.reactive.server.core.parameters.converters;
+
+public class CharacterParamConverter implements ParameterConverter {
+
+    @Override
+    public Object convert(Object parameter) {
+        String str = parameter.toString();
```

Comment:
This should allow `null` values, no?

Reply:
`null` doesn't reach this point AFAIK

Reply:
Just checked, the other converters don't check for null either.